tests/test_return_code.py

```diff
@@ -15,6 +15,8 @@
 PASSED = 0
 # Failure indicates the QA tool reported some issues.
 FAILED = 1
+# pylint exits with 20 instead of 1
+PYLINT_FAILED = 20
```

Comment:
Given we are starting to see different exit codes returned by these tools, I am suggesting to check `assert pylint_runner([DIRTY_NOTEBOOK]) != PASSED` for these scenarios. No need to add new exit codes in the future and so the tests would not break.